### PR TITLE
Add DTOs

### LINKED ISSUE #27: Create DTOs

### PR DESCRIPTION
**Pull request by SharifovDeveloper**

### CODE REVIEW

File: StaySphere.Api/StaySphere.Domain/DTOs/Booking/BookingForUpdateDto.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace StaySphere.Domain.DTOs.Booking
+{
+    public record BookingForUpdateDto(
+        int Id,
+        int GuestId,
+        int EmployeeId,
+        int RoomId);
+
```
Comment (Nabiddinov) [inline StaySphere.Api/StaySphere.Domain/DTOs/Booking/BookingForUpdateDto.cs:8]: bo'sh joy qolib ketmasin,keraksiz usinglarni olib tashlang

File: StaySphere.Api/StaySphere.Domain/DTOs/Category/CategoryForCreateDto.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StaySphere.Domain.DTOs.Category
+{
+    public record CategoryForCreateDto(string Name);
+
```
Comment (Nabiddinov) [inline StaySphere.Api/StaySphere.Domain/DTOs/Category/CategoryForCreateDto.cs]: keraksiz qatorni olib tashlang

File: StaySphere.Api/StaySphere.Domain/DTOs/Document/DocumentDto.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿using StaySphere.Domain.DTOs.Guest;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (Nabiddinov) [inline StaySphere.Api/StaySphere.Domain/DTOs/Document/DocumentDto.cs]: keraksiz usinglarni olib tashlang

File: StaySphere.Api/StaySphere.Domain/DTOs/Booking/BookingDto.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+﻿using StaySphere.Domain.DTOs.Review;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```
Comment (Nabiddinov) [inline StaySphere.Api/StaySphere.Domain/DTOs/Booking/BookingDto.cs]: keraksiz usinglarni olib tashlang

**Review by Nabiddinov — APPROVED** [review]
good

File: StaySphere.Api/StaySphere.Domain/DTOs/Document/DocumentDto.cs (hunk 3 above)
Comment (SharifovDeveloper) [reply StaySphere.Api/StaySphere.Domain/DTOs/Document/DocumentDto.cs]: Corrected code

File: StaySphere.Api/StaySphere.Domain/DTOs/Booking/BookingDto.cs (hunk 4 above)
Comment (SharifovDeveloper) [reply StaySphere.Api/StaySphere.Domain/DTOs/Booking/BookingDto.cs]: Ok

**Review by diyorzz — APPROVED** [review]

File: StaySphere.Api/StaySphere.Domain/DTOs/Booking/BookingDto.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
+﻿using StaySphere.Domain.DTOs.Review;
+
+namespace StaySphere.Domain.DTOs.Booking
+{
+    public class BookingDto
+    {
+        public int Id { get; set; }
+
```
Comment (Mirazyzz) [inline StaySphere.Api/StaySphere.Domain/DTOs/Booking/BookingDto.cs:8]: redundant empty line.

**Comment by Mirazyzz** [comment]
Why are you using type data types for DTOs one is record another is class? Would it be better to choose only one?

**Review by Mirazyzz — COMMENTED** [review]
Why are you using type data types for DTOs one is record another is class? Would it be better to choose only one?

**Pull request merged**